lecture/urls.py

```
@@ -1,9 +1,10 @@
 from django.urls import path
-from . import views 
+from . import views
 from django.conf.urls.static import static
```

Ruff (0.8.2)

`django.conf.urls.static.static` imported but unused

Remove unused import: `django.conf.urls.static.static`

(F401)